### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by chenkennt**
Add default issue template.
@vicancy @vwxyzh @superyyrrzz @qinezh

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+e.g. Long polling transport tries reconnecting forever when ping succeeds but poll request fails*
+
+### Functional impact
+*Does the bug result in any actual functional issue, if so, what?*  
+e.g. If poll request starts working again, it recovers. Otherwise, it sends many requests per sec to the servr.
+
+### Minimal repro steps
+*What is the smallest, simplest set of steps to reproduce the issue. If needed, provide a project that demonstrates the issue.*
+
+1. Enable SQL scale out in the samples app
```
Comment (vwxyzh) [inline ISSUE_TEMPLATE.md]: prepend *e.g.*?

**Comment by vwxyzh** [comment]
👍

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+### Title
+*A short description of the bug that becomes the issue title*  
+e.g. Long polling transport tries reconnecting forever when ping succeeds but poll request fails*
+
+### Functional impact
+*Does the bug result in any actual functional issue, if so, what?*  
+e.g. If poll request starts working again, it recovers. Otherwise, it sends many requests per sec to the servr.
```
Comment (qinezh) [inline ISSUE_TEMPLATE.md]: servr -> server

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+### Title
+*A short description of the bug that becomes the issue title*  
+e.g. Long polling transport tries reconnecting forever when ping succeeds but poll request fails*
```
Comment (qinezh) [inline ISSUE_TEMPLATE.md]: remove the `*` at end of the line?

**Comment by superyyrrzz** [comment]
LGTM

**Comment by qinezh** [comment]
:+1:

File: ISSUE_TEMPLATE.md (hunk 3 above)
Comment (hellosnow) [inline ISSUE_TEMPLATE.md]: ”poll request“ should be "pull request"

**Pull request merged**